scripts/jenkins/qat/tc-deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Deploys current jenkins-build test image for dpdk-app (crypto-perf)
+# in the cluster.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+REPO_ROOT=$(realpath $(dirname $0)/../../..)
+for test in $(ls ${REPO_ROOT}/deployments/qat_dpdk_app/ | grep test-*); do
+  kubectl apply -k ${REPO_ROOT}/deployments/qat_dpdk_app/${test}/ &
```

Comment:
`&` needed?

Reply:
We can deploy all in sequence as well.. but deploying in background we can save some secs if test cases get more in number in the future.